src/parser.rs

```diff
@@ -153,7 +154,11 @@ impl Library {
                     fields.push(f);
                 })
             }
-            "virtual-method" => parser.ignore_element(),
+            "virtual-method" => {
+                self.read_signal(parser, ns_id, elem).map(|v| {
```

Comment:
Yes, except that they might have annotations for the parameters and return value

Reply:
I figured. I'll get to that later. You have to have a fair amount of knowledge of gobject/gir internals here. Which I don't have :D.